root/static/scripts/edit/externalLinks.js

```diff
@@ -705,6 +705,15 @@ class _ExternalLinksEditor
                     Did you mean to paste something else?`),
         target: URLCleanup.ERROR_TARGETS.URL,
       };
+    } else if (isNewOrChangedLink && isCritiqueBrainz(link.url)) {
+      error = {
+        message: texp.l(
+          `Please don’t enter CritiqueBrainz links — reviews
+           are automatically linked from the “{reviews_tab_name}” tab.`,
+          {reviews_tab_name: l('Reviews')},
+        ),
```

Comment:
> I expected that doing the string like that will ensure that the Reviews tab name uses whatever translation we already have, but I tried with Italian and it still showed "Reviews" despite us having a translation - is my understanding of how it should work wrong, or does it only work if the main string is translated? :)

Your understanding is correct. I tested translating the new string and updating POT/PO files locally and “Reviews” remains untranslated here.

My guess is that it has something to do with component hydration. I didn’t find other cases of `l` being used in arguments of `texp.l` under `root/static/scripts`. It might be an undocumented use case if not a bug.